lib/charms/haproxy/v0/haproxy_route_tcp.py

```diff
@@ -0,0 +1,1738 @@
+# pylint: disable=too-many-lines,duplicate-code
+"""Haproxy-route interface library.
+
+## Getting Started
+
+To get started using the library, you just need to fetch the library using `charmcraft`.
+
+```shell
+cd some-charm
+charmcraft fetch-lib charms.haproxy.v1.haproxy_route_tcp
```

Comment:
v0

Reply:
I took note of this and will create a PR in haproxy to update the library